tests/acceptance/email-change-test.js

```diff
@@ -185,7 +185,7 @@ module('Acceptance | Email Change', function (hooks) {
 
       await click('[data-test-email-input] [data-test-resend-button]');
       assert.dom('[data-test-email-input] [data-test-resend-button]').isEnabled().hasText('Resend');
-      assert.dom('[data-test-email-input] [data-test-error]').hasText('Unknown error in resending message');
+      assert.dom('[data-test-email-input] [data-test-error]').hasText('Error in resending message: [object Object]');
```

Comment:
This is not exactly pretty, but I didn't touch the formatting code, I just ensured that we actually return an Ember Data error from the method call. I have another PR queued up already that fixes the formatting and error display.